spec/routes/web/inference_api_key_spec.rb

```diff
@@ -41,14 +41,14 @@
     it "inference api key page allows removing inference api keys" do
       btn = find(".delete-btn")
       data_url = btn["data-url"]
-      _csrf = btn["data-csrf"]
-      page.driver.delete data_url, {_csrf:}
+      csrf = btn["data-csrf"]
+      page.driver.delete data_url, {_csrf: csrf}
```

Comment:
Do we believe this makes the code better?  It's obvious the reason it was done previously was for hash value omission, which seems like a reasonable reason to allow it.

Reply:
Makes sense
https://github.com/ubicloud/ubicloud/pull/3046/commits/551c3c5f54ce5fcbf496b26df067c852c7c66ef9